public-ui/src/app/data-browser/views/ehr-view/ehr-view-react.component.tsx

```diff
@@ -0,0 +1,650 @@
+                        </div>
+                        <div className='tbl-d body-lead'> Participants of {totalParticipants.toLocaleString()}
+                                      <TooltipReactComponent tooltipKey='participantCountHelpText' label='EHR Tooltip Hover'
+                                                         searchTerm={searchWord} action='Participant count tooltip hover' />
+                        </div>
+                        <div className='tbl-d body-lead'> % of {totalParticipants.toLocaleString()}
+                          <TooltipReactComponent tooltipKey='percentageOfParticipants'
+                                             label='EHR Tooltip Hover' searchTerm={searchWord}
+                                             action='Percentage of participant count tooltip hover' />
+                        </div>
+                        {(domain.domain.toLowerCase() === 'measurement') &&
+                        <div className='tbl-d body-lead info-text'>
+                          Data Type
+                          <div className={dropdownClass}>
+                            <button className='dropdown-toggle'>
+                              <ClrIcon shape='filter-grid' className={filterIconClass} onClick={() =>
+                              this.setState({selectedMeasurementTypeFilter: !selectedMeasurementTypeFilter})} />
+                            </button>
+                            <div className='dropdown-menu'>
+                                <div className='clr-checkbox-wrapper'>
+                                    <div className='checkbox-input'>
+                                        <input type='checkbox' id='checkbox1' className='clr-checkbox' onClick={() =>
+                                        this.setState({measurementTestFilter: !measurementTestFilter},
+                                        () => {this.getDomainTotals(); this.getTopConcepts(); } )} defaultChecked={measurementTestFilter}/>
+                                        <label htmlFor='checkbox1' className='checkbox-label'><i className='fas fa-vial fa-rotate-45'
+                                        style={{'transform': 'rotate(315deg)'}}></i> Tests</label>
+                                    </div>
+                               </div>
+                               <div className='clr-checkbox-wrapper'>
+                                    <div className='checkbox-input'>
+                                        <input type='checkbox' id='checkbox2' className='clr-checkbox' onClick={() =>
+                                        this.setState({measurementOrderFilter: !measurementOrderFilter},
+                                        () => {this.getDomainTotals(); this.getTopConcepts(); } )} defaultChecked={measurementOrderFilter}/>
+                                        <label htmlFor='checkbox2' className='checkbox-label'>
+                                        <i className='far fa-file-signature'></i> Orders</label>
```

Comment:
Nit: use self-closing syntax for empty `<i>` tag

Reply:
changed this thanks.